Dockerfile

```diff
@@ -38,7 +37,26 @@ COPY --from=voila_rootfs_builder /usr/ /usr/
 COPY --from=voila_rootfs_builder /etc/ /etc/
 
 
-FROM base_image_python AS kernel_rootfs_builder
+FROM base_image AS kernel_rootfs_builder
+RUN apt-get update && \
+	apt-get install -y --no-install-recommends wget && \
+	apt-get install -y --no-install-recommends ca-certificates
+# Miniconda
+ARG conda_dir="/usr/miniconda3"
+ENV PATH="${conda_dir}/bin:${PATH}"
+ARG PATH="${conda_dir}/bin:${PATH}"
+ARG mconda="Miniconda3-py38_4.12.0-Linux-x86_64.sh"
+RUN wget \
+    https://repo.anaconda.com/miniconda/"$mconda" && \
+    bash "$mconda" -p "$conda_dir" -b && \
+    rm -f "$mconda"
+# Create conda env
+ARG cenv="customer-env.yml"
+COPY "$cenv" .
+RUN conda env create -n voilaenv --file "$cenv"
+# Use conda env for subsequent RUNs
+SHELL ["conda", "run", "-n", "voilaenv", "/bin/bash", "-c"]
+
 RUN python3 -m pip install --no-cache-dir altair bqplot ipykernel ipyvolume ipywidgets pandas perspective-python==1.0.1 pyarrow PyYAML quilt3 scipy
```

Comment:
Shouldn't we move this deps to customer-env.yml?

Reply:
We should but solver is super duper slow (30 min) and will get slower so I'd rather not touch it